File: WebApp/src/app/cadastros/models/pagamento.ts
import { FormaPagamento } from "./formaPagamento";
import { Conta } from "./conta";

export class Pagamento {
    cdPagamento: number;
    cdFormaPagamento: number;
    formaPagamento: FormaPagamento;
    cdConta: number;
    conta: Conta;
    vlPago: number;
    fgPago: boolean;
    nrParcela: number;
    vlParcela: number;
    dtBaixa: Date;
    dtPagamento: Date;
    dtRegistro: Date;
    dtAlteracao: Date;
}

File: BikeManagerAPI/BikeManagerAPI/Models/Pagamento.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;

namespace BikeManagerAPI.Models
{
    public partial class Pagamento
    {
        public int CdFormapagamento { get; set; }
        public int CdConta { get; set; }
        public decimal VlPago { get; set; }
        public DateTime DtPagamento { get; set; }
        public DateTime DtRegistro { get; set; }
        public DateTime? DtAlteracao { get; set; }
        public Conta Conta { get; set; }
        public FormaPagamento FormaPagamento { get; set; }
    }
}
